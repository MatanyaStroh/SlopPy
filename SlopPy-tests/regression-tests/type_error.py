x = 5 + 'foo'
assert type(x) is NA
