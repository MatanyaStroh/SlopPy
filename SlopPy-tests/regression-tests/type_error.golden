{NA <type 'exceptions.TypeError'> "unsupported operand type(s) for +: 'int' and 'str'"
 Traceback (most recent call last):
  File "regression-tests/type_error.py", line 1, in <module>
    x = 5 + 'foo'
}
